.github/workflows/build.yml

```diff
@@ -213,3 +214,42 @@ jobs:
         yarn storybook:deploy
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  publish:
+      runs-on: ubuntu-latest
+      if: ${{ github.ref == 'refs/heads/master' }}
+      steps:
+        # Checkout the Repo
+        - uses: actions/checkout@v3
+        # Setup Node 16
+        - uses: actions/setup-node@v3
+          with:
+            node-version: 16
+            cache: 'yarn'
+        # Restore node_modules - Cache should exist as one was created in previous 'install' job
+        - name: Restore Cache -  Node Modules
+          id: cache-node-modules
+          uses: actions/cache@v2
+          with:
+            path: '**/node_modules'
+            key: node-modules-${{ hashFiles('**/yarn.lock') }}
+        # Run 'yarn' if cache doesn't exist. Use --prefer-offline to download packages from yarn cache folder where possible
+        - name: Install Dependencies
+          if: steps.cache-node-modules.outputs.cache-hit != 'true'
+          run: yarn --prefer-offline
+        # Restore Cache - Turborepo Dist
+        - name: Cache Turborepo Dist
+          id: turborepo-dist-cache
+          uses: actions/cache@v2
+          with:
+            path: .turbo
+            key: turborepo-dist-cache-${{ github.ref }}-${{ github.sha }}
+            # Use one of the following keys if the above is not found.
+            restore-keys: |
+              turborepo-dist-cache-${{ github.ref }}
+              turborepo-dist-cache-
+        - run: npm test
```

Comment:
Might be worth changing this to `yarn build`

I think even though we add the turborepo cache, `yarn build` gets turborepo to move the `dist` folders to the correct place.

Reply:
sorry - which lines specifically should I change to yarn build?